src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (value) => {
```

Comment:
Ensure that the parameter `value` is always a number when provided. Currently, the function will incorrectly add non-number values to `sum` if they are provided. You might want to add a type check to ensure that `value` is a number before adding it to `sum`.